models/utility.models.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection.js");
+const format = require("pg-format");
+
+exports.checkColExists = (table, column, value) => {
+    const sqlString = format(
+      "SELECT * FROM %I WHERE %I = %L",
+      table,
+      column,
+      value
+    );
+    return db.query(sqlString).then(({ rows: values }) => {
+      if (!values.length) {
+        return Promise.reject({ status: 404, msg: "not found" });
+      }
+    });
+  };
```

Comment:
Love the generalised nature of this function - love that it is extracted into its own file.
I would suggest 2 things:
- `checkColExists` could be renamed - it's not checking if a column exists, rather a value in a particular column
- If you are now using `pg-format` in your actual application rather than just seeding, make sure it is no longer listed as a DEV dependency